packages/zevm-app-contracts/contracts/xp-nft/xpNFT_V2.sol

```diff
@@ -16,13 +16,19 @@ contract ZetaXP_V2 is ZetaXP {
     }
 
     mapping(uint256 => uint256) public levelByTokenId;
+
+    // Event for Level Set
     event LevelSet(address indexed sender, uint256 indexed tokenId, uint256 level);
 
     function version() public pure override returns (string memory) {
         return "2.0.0";
     }
 
-    function _verifySetLevelSignature(SetLevelData memory data) private view {
+    function _verifyUpdateNFTSignature(uint256 tokenId, UpdateData memory updateData) internal view {
+        _verify(tokenId, updateData);
+    }
```

Comment:
Do we need to modify this?

Reply:
it's not a modify, but looks like in git. We add a new one then git thinks we are updating and after adding other. Take a look to the next line and it's there